website_sale_product_brand/views/product_brand.xml

```diff
@@ -38,25 +38,19 @@
                         <t t-if="brand_rec">
                             <div class="row">
                                 <t t-foreach="brand_rec" t-as="o">
-                                    <div class="col-lg-3 d-lg-inline-block">
+                                    <div
+                                        class="card border-0 col-lg-3 col-md-6 mb-4 text-center brand_item"
+                                    >
                                         <a
                                             t-att-href="keep('/shop/brands',brand = o.id)"
                                         >
-                                            <div class="row text-center">
-                                                <div class="col">
-                                                    <img
-                                                        itemprop="image"
-                                                        width="200px"
-                                                        height="200px"
-                                                        class="img"
-                                                        t-attf-src="/website/image/product.brand/#{o.id}/logo#{'?max_width=300&amp;max_height=300'}"
-                                                    />
-                                                </div>
-                                            </div>
-                                            <div class="row text-center">
-                                                <div class="col">
-                                                    <t t-esc="o.name" />
-                                                </div>
+                                            <img
+                                                class="card-img-top img-fluid brand_item_image"
+                                                t-attf-src="/website/image/product.brand/#{o.id}/logo"
```

Comment:
Wouldn't it be enough to use img-fluid and tell the controller to give us a 200x200 image?
```suggestion
                                                t-attf-src="/website/image/product.brand/#{o.id}/logo/200x200"
```

Reply:
It doesn't work as that doesn't return a 200x200 canvas if the image isn't 1:1